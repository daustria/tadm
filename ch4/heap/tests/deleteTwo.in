i 5
i 3
i 1
i 7
i 9
i 10
d
p
q
