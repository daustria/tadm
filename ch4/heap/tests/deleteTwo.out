3 5 10 7 9 
